### PR TITLE
Use newest cartridge for tests

### PR DESCRIPTION
**Pull request by dokshina**

### CODE REVIEW

File: create-packages.sh [hunk 1]
```diff
@@ -21,7 +21,10 @@ version=1.0.0-0
 pack_flags=''
 if [[ $(tarantool -V) == *"Target: Darwin"* ]]; then
     pack_flags+=' --use-docker'
-    if [ -z "${TARANTOOL_SDK_PATH}" ]; then
+fi
+
+if [[ $(tarantool -V) == *"Tarantool Enterprise"* ]]; then
```
Comment (hackallcode) [inline create-packages.sh]: Why if you have Tarantool Enterprise, you should set TARANTOOL_SDK_PATH?

Comment (dokshina) [reply create-packages.sh]: I've fixed that. If you run this script on OS X and use TE, your package will contain your current SDK (which is sdk-macos and doesn't work on Linux) or the one specified in TARANTOOL_SDK_PATH (you should specify here Linux SDK).

Comment (hackallcode) [reply create-packages.sh]: Okay, now it's better. But why you don't want to use "and" operation instead 2 "if"?

Comment (dokshina) [reply create-packages.sh]: Fixed.

**Review by hackallcode — APPROVED** [review]

**Pull request merged**